### PR TITLE
MINOR: fix comment in ClusterConnectionStates

### PR DESCRIPTION
**Pull request by yeoncheol-jang**
Found minor typo in ClusterConnectionStates.java

### More detailed description of your change
`iff` --> `if`

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by yeoncheol-jang** [comment]
Call for review @dajac  @junrao  @guozhangwang  @ijuma

File: clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public ClusterConnectionStates(long reconnectBackoffMs, long reconnectBackoffMax
     }
 
     /**
-     * Return true iff we can currently initiate a new connection. This will be the case if we are not
+     * Return true if we can currently initiate a new connection. This will be the case if we are not
```
Comment (junrao) [inline clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java:71]: iff means if and only if. So, this seems correct?

**Review by junrao — COMMENTED** [review]
@YeonCheolGit : Thanks for the PR. Left a comment below.

File: clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java (hunk 1 above)
Comment (yeoncheol-jang) [reply clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java:71]: @junrao 
Oh really didn't know about that. Sorry for interrupt you and thank you for letting me know:) 
I will close this PR.

**Pull request closed without merging**